src/store/proposals/proposals.sagas.ts

```diff
@@ -108,10 +115,69 @@ const watchProposalsList = createWatcherSaga({
   stopAction: proposalsActions.stopWatchingProposalsList.type,
 });
 
+export function* addProposal({ payload }: PayloadAction<AddProposalFields>) {
+  try {
+    const account = yield* select(accountSelector);
+    const staking = yield* select(stakingContractSelector);
+    const isGovAdmin =
+      payload[AddProposalInputs.SendProposalContract] ===
+      GOVERNANCE_OPTIONS.GOVERNER_ADMIN.id;
+
+    const govSelector = isGovAdmin
+      ? governorAdminSelector
+      : governorOwnerSelector;
+
+    const governor = yield* select(govSelector);
+
+    if (!account || !governor || !staking) {
+      throw new Error('Wallet not connected');
+    }
+
+    const threshold = yield* call(governor.proposalThreshold);
+    const votes = yield* call(staking.getCurrentVotes, account);
+
+    if (threshold.gt(votes)) {
+      const errorMsg = `Your voting power must be at least ${threshold.toString()} to make a proposal`;
```

Comment:
The `toString()` here is unnecesary. JS will implicitly invoke `toString()` when using template strings
```suggestion
      const errorMsg = `Your voting power must be at least ${threshold} to make a proposal`;
```